client/data/newsletter-categories/test/use-mark-as-newsletter-category-mutation.test.tsx

```diff
@@ -73,7 +73,9 @@ describe( 'useMarkAsNewsletterCategoryMutation', () => {
 			await result.current.mutateAsync( categoryId );
 		} );
 
-		expect( invalidateQueriesSpy ).toHaveBeenCalledWith( [ 'newsletter-categories', 123 ] );
+		expect( invalidateQueriesSpy ).toHaveBeenCalledWith( {
+			queryKey: [ 'newsletter-categories', 123 ],
```

Comment:
Not blocking: This entire test seems to be testing internal RQ functionality, so it might be a good reason to re-evaluate which tests are unnecessary and potentially deletable.